jarbas/chamber_of_deputies/views.py

```diff
@@ -77,25 +44,37 @@ def _build_filters(self):
         if in_latest is not None:
             self.queryset = self.queryset.in_latest_dataset(in_latest)
 
+        # filter search_vector
+        vector_param = self._vector_param()
+        if vector_param:
+            self.queryset = self.queryset.search_vector(vector_param)
+
         # filter queryset
         if filters:
             self.queryset = self.queryset.tuple_filter(**filters)
 
-    def _build_vector_filters(self):
-        """ 
-            Builder and aplly all vector filters on QuerySet 
-        """
+        # change ordering if needed
+        order_by = self.request.query_params.get('order_by')
+        if order_by == 'probability':
+            self.queryset = self.queryset.order_by_probability()
```

Comment:
We might want to sort by `rank` if:

* `search` parameter is in use
* We implemented it in the `search_vector` queryset
*  User's passed no `order_by`

What do you think?